src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -144,30 +145,37 @@ def _verify_get_params_dict(params_dict):
 
 
 def _build_output_tables(n_iters_run, model_table, args, transformer, **kwargs):
-    if transformer is None:
+    # transformer should always be a valid object
+    original_table = transformer.original_table
+    if not original_table:
```

Comment:
As far as I understand, `original_table` will always be a dict object such that `not original_table` will always return `false`. Let us remove the `if` branch if that is case.